src/components/SolidLoginForm.jsx

```diff
@@ -0,0 +1,59 @@
+import { getDefaultSession, fetch } from "@inrupt/solid-client-authn-browser";
+import {  getLiteral, getProfileAll, getThing } from "@inrupt/solid-client";
+import { FOAF } from "@inrupt/vocab-common-rdf";
+import { useState } from "react";
+
+function SolidLoginForm(props) {
+  const session = getDefaultSession();
+  let webId = session.info.webId
+  const [name, setName] = useState(undefined)
+  if (webId) {
+    getProfileAll(webId, { fetch: fetch }).then((dataSet) => {
+      let profile = dataSet.webIdProfile
+      const webIdThing = getThing(profile, webId)
+      setName(getLiteral(webIdThing, FOAF.name).value) 
+    });
+  }
+
+  function handleLogin(event) {
```

Comment:
Missing documentation.

Reply:
I have added documentation for this function in [commit 62fff5f](https://github.com/SolidLabResearch/generic-data-viewer/pull/23/commits/62fff5f053c921f765cdca30902d18797743f9f4)